.storybook/preview.js

```diff
@@ -19,6 +20,8 @@ export const parameters = {
   },
 }
 
+const firestoreResponseMock = { get: () => true }
```

Comment:
Pour mieux refléter ce que renvoie `get` ?
```suggestion
const firestoreResponseMock = { get: () => ({}) }
```